R/impact_central.R

```diff
@@ -496,7 +493,7 @@ impact_by_year_of_vaccination_activity_type <- function(
     dplyr::inner_join(impact_ratio, by = c("country", "activity_type")) %>%
     dplyr::mutate(impact = impact_ratio * fvps) %>%
     dplyr::select(country, vaccine, activity_type, year, burden_outcome,
-                  impact) %>%
+                  impact, fvps) %>%
```

Comment:
Where are these used? Are the fvps needed by some report?

Reply:
We need impact_ratios and fvps in our reports for funders. I picked up fvps, but maybe impact_ratio is a better choice. fvps can be attached at a later stage, but impact ratio will be lost if not in the output.